lib/virt_autotest/common.pm

```diff
@@ -131,12 +145,12 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             delete $guests{$guest} unless grep { $_ eq $guest } @allowed_guests;
         }
     } elsif (is_sle('=15-SP5')) {
-        my @allowed_guests = qw(sles12sp5HVM sles12sp5PV sles15sp5HVM sles15sp5PV sles15sp6HVM sles15sp6PV);
+        my @allowed_guests = qw(sles12sp5HVM sles12sp5PV sles12sp5ltssesHVM sles12sp5ltssesPV sles15sp5HVM sles15sp5PV sles15sp6HVM sles15sp6PV);
```

Comment:
As stated in the ticket, only 12sp5 ltss-es host has test for  12sp5 ltss-es guest, any other host does not need to test 12sp5 ltss-es guest.